src/main.rs

```diff
@@ -126,5 +133,8 @@ fn main() {
                     .setup(),
                 )
         }
+        Commands::Sleep => loop {
+            sleep(Duration::from_secs(3600));
+        },
```

Comment:
Any chance we'd want to just add a sleep or pause flag that runs after the other commands so we can have a single container? That might be an anti-pattern not sure.

Reply:
We would need to add something to handle readiness checks (ie: a TCP listener?). Having the actual setup in an init container lets us ensure that the disks are set up before the pod gets marked as ready, without actually needing any readiness checks.